tv_apps/smart-home/test/unit/mock_spatial_navigator.js

```diff
@@ -0,0 +1,22 @@
+ /* globals evt */
+
+(function(exports) {
+  'use strict';
+
+  var MockSpatialNavigator = function() {
+  };
+
+  MockSpatialNavigator.prototype = evt({
+    init: function() {},
+    focus: function() {},
+    //remove: function() {},
+    getFocusedElement: function() {
+      return this.m_focusedElement;
+    },
+    remove: function(node) {},
+
+    m_focusedElement: null
+  });
+
+  exports.MockSpatialNavigator = MockSpatialNavigator;
+})(window);
```

Comment:
We already have a `mock_spatial_navigator.js` in `shared/test/unit/mocks/smart-screen/`. I think we should consolidate these two `mock_spatial_navigator.js`. And since other apps might need `mock_spatial_navigator.js` to run tests, I think it is better to place it under shared/